NYTimesMostViewedArticles/API/ArticleAPI.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ArticleAPI.swift
+//  NYTimesMostViewedArticles
+//
+//  Copyright © 2020 Sagar Pahwa. All rights reserved.
+//
+
+import Foundation
+
+protocol ArticlesAPIType {
+    func getMostViewedArticles(period: PeriodSection, completion: @escaping (Response?, Error?)->())
+}
+
+extension ArticlesAPIType {
+    static func url(period: PeriodSection) -> URL {
+        let string = "\(API.baseURL)/\(API.contentURL)/\(period.rawValue).json?api-key=\(Keys.nyTimesKey.rawValue)"
+        return URL(string: string)!
+    }
+}
+
+class ArticlesAPI: ArticlesAPIType {
```

Comment:
This is more of a service class, please follow proper naming convention that should be self-descriptive.
Hence please rename it to MostPopularArticlesNetworkService